python_modules/libraries/dagster-fivetran/dagster_fivetran/assets.py

```diff
@@ -0,0 +1,99 @@
+from typing import List, Optional
+
+from dagster import AssetKey, Out, Output
+from dagster.core.asset_defs import AssetsDefinition, multi_asset
+from dagster.utils.backcompat import experimental
+from dagster_fivetran.resources import DEFAULT_POLL_INTERVAL
+from dagster_fivetran.utils import generate_materializations
+
+
+@experimental
+def fivetran_assets_factory(
```

Comment:
For this kind of function, I think it's preferable for the name to center on a verb, e.g. `build_fivetran_assets`. "factory" would be appropriate for a class with a method that builds assets.